java/src/jmri/jmrit/roster/RosterEntry.java

```diff
@@ -599,8 +599,8 @@ public void setDateModified(@Nonnull String date) throws ParseException {
                 try {
                     setDateModified(customFmt.parse(date));
                 } catch (ParseException ex3) {
-                    // then try with a specific format to handle e.g. "01-Oct-2016 9:13:36"
-                    customFmt = new SimpleDateFormat("dd-MMM-yyyy hh:mm:ss");
+                    // then try with a specific format to handle e.g. "01-Oct-2016 21:13:36"
+                    customFmt = new SimpleDateFormat("dd-MMM-yyyy HH:mm:ss");
```

Comment:
@bobjacobsen 
Could this break anything?

Reply:
I think this direction (12 hours to 24 hours) is a safe forward-going change.  In 12-hour time, "10" is ambiguous, so this change will make it definitely AM.